flutter-for-android.md

```diff
@@ -0,0 +1,1845 @@
+---
+layout: page
+title: Flutter for Android Developers
+permalink: /flutter-for-android/
+---
+This document is meant for Android developers looking to transfer their existing Android knowledge to build mobile apps in Flutter. If you understand the fundamentals of the Android framework then you can use this document as a jump start to Flutter development.
+
+This document can be used as a cookbook by jumping around and finding questions that are most relevant to your needs.
+
+# Views
+
+**What is the equivalent of a View in Flutter?**
+
+In Android, the View is the foundation of everything that shows up on the 
+screen. From Buttons, Toolbars, and Inputs, everything is a View.  In Flutter 
+the equivalent of a View would be Widget. However, Widgets have a few 
+differences when compared with a View. To start, widgets only last for a frame, 
+and on every frame, we create a tree of widget instances. In comparison, on 
+Android when a View is drawn it does not redraw until invalidate is called. 
+
+This is because unlike Android’s view hierarchy system where we have Views and 
```

Comment:
this is a long sentence. Can you break it into two?

Reply:
Fixed